config/default/manager_webhook_patch.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: controller-manager
+  name: controller-manager-webhook
```

Comment:
shouldn't this be submariner-operator-webhook then?

Reply:
Same here this webhook is not part of our build. we are not using webhooks at this moment, maybe in the future.